python/tk_multi_snapshot/snapshot.py

```diff
@@ -432,7 +432,7 @@ def show_snapshot_dlg(self):
 
             # try to launch "Shotgun Save As" command if we have it:
             save_as_cmd = tank.platform.current_engine().commands.get(
-                "Shotgun Save As..."
+                "ShotGrid Save As..."
```

Comment:
Please also look for `Shotgun Save As...` in case the client hasn't updated the workfiles app.